packages/compass/src/app/styles/apply-global-styles.ts

```diff
@@ -0,0 +1,45 @@
+import createCache from '@emotion/cache';
```

Comment:
Since we're using the `@emotion/css` library from leafygreen: https://github.com/mongodb/leafygreen-ui/blob/main/packages/emotion/src/index.ts#L15 we don't have access to the `Global` item as it is react specific. 

Under the hood it's doing a similar thing to what we are here: https://github.com/emotion-js/emotion/blob/main/packages/react/src/global.js#L62

Reply:
>  Edit the body element to ensure the entire page styles nicely

Yeah, you're right we would need it at some point anyway

>  I can add a component like that to Compass home and move these styles there - how do you feel about that?

That makes sense to me

> we don't have access to the Global item as it is react-specific

Ah I see, that's inconvenient yeah, what is the difference between doing `from '@emotion/cache` vs `from '@emotion/react'`?

Reply:
Removed the emotion package usage, went with a simpler approach using a one time styles `injectGlobal` from leafygreen's emotion.